examples/website-demos/social-messaging/src/components/Message/SocialMessageUI.tsx

```diff
@@ -99,27 +148,30 @@ export const SocialMessage: React.FC<
   return (
     <div className={`message-wrapper ${myMessage ? 'right' : ''}`}>
       {!myMessage && <Avatar size={36} image={message.user?.image} />}
-      <div className='message-wrapper-inner'>
-        {showDetailedReactions && isReactionEnabled && (
-          <ReactionSelector ref={reactionSelectorRef} />
-        )}
-        {hasReactions && !showDetailedReactions && isReactionEnabled && <SimpleReactionsList />}
+      <div className={`message-wrapper-inner ${myMessage ? 'my-message' : ''}`}>
+        {hasReactions && isReactionEnabled && <SimpleReactionsList />}
         <MessageText customWrapperClass={`${myMessage ? 'my-message' : ''}`} />
         {message.attachments?.length ? <Attachment attachments={message.attachments} /> : null}
-        <MessageOptions displayLeft={false} displayReplies={true} />
-        <MessageRepliesCountButton reply_count={message.reply_count} />
-        <div className={`message-wrapper-inner-data ${myMessage ? 'my-message' : ''}`}>
-          {!myMessage && (
-            <div className='message-wrapper-inner-data-info'>
-              {message.user?.name || message.user?.id}
-            </div>
-          )}
-          {myMessage &&
-            message.status === 'received' &&
-            readByMembers &&
-            readByMembers?.length < 1 && <DeliveredCheckmark />}
-          {myMessage && readByMembers && readByMembersLength && <DoubleCheckmark />}
-          <MessageTimestamp customClass='message-wrapper-inner-data-time' />
+        {showDetailedReactions && isReactionEnabled && (
+          <ReactionSelector reactionOptions={customReactions} ref={reactionSelectorRef} />
+        )}
+        <ThreadReply />
+        <div className='message-wrapper-inner-options'>
+          {myMessage && <MessageOptions displayReplies={true} />}
+          <div className='message-wrapper-inner-data'>
+            {myMessage &&
+              message.status === 'received' &&
+              readByMembers &&
+              readByMembers?.length < 1 && <DeliveredCheckmark />}
+            {myMessage && readByMembers && readByMembersLength && <DoubleCheckmark />}
+            {!myMessage && (
+              <div className='message-wrapper-inner-data-info'>
+                {message.user?.name || message.user?.id}
+              </div>
+            )}
+            <MessageTimestamp customClass='message-wrapper-inner-data-time' />
+          </div>
+          {!myMessage && <MessageOptions />}
```

Comment:
do we need two references to MessageOptions? i'm assuming this is absolutely positioned so can't we do...
`<MessageOptions displayReplies={myMessage} />`

Reply:
there was no need for 2 - I removed 1 and adjusted with css